packages/css-data/src/parse-css.ts

```diff
@@ -57,10 +57,24 @@ const parseCssValue = (
   const expanded = new Map(expandShorthands([[property, value]]));
   const final = new Map();
   for (const [property, value] of expanded) {
-    final.set(
-      property,
-      parseCssValueLonghand(property as StyleProperty, value)
-    );
+    let finalProperty = property as StyleProperty;
+    // Appearance is baseline since 2022 https://developer.mozilla.org/en-US/docs/Web/CSS/appearance
+    if (property === "-webkit-appearance") {
```

Comment:
It's not the only case. I see in presets -webkit-transform, -webkit-user-select, -moz-user-select, even -mos-placeholder. A lot of super old stuff.